src/data/leo-images.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  Default: { // TODO: these names will include some installed packages
+    version: 'FINAL',
+    updated: '2019-08-26',
+    packages: 'https://storage.googleapis.com/terra-docker-image-documentation/terra-jupyter-base-0.0.1-python-packages.txt', // TODO: not the real URL
+    image: 'us.gcr.io/broad-dsp-gcr-public/leonardo-jupyter:5c51ce6935da'
```

Comment:
Is this a standard pattern for image URLs, or should there be a protocol specifier on the front?

Reply:
Nope, there's no protocol specifier.